ta/arch/arm/link.mk

```diff
@@ -4,13 +4,14 @@ link-script = $(TA_DEV_KIT_DIR)/src/user_ta_elf_arm.lds
 link-script-pp = $(link-out-dir)/ta.lds
 
 FIX_TA_BINARY = $(TA_DEV_KIT_DIR)/scripts/fix_ta_binary
-
+SIGN = $(TA_DEV_KIT_DIR)/scripts/sign.py
+TA_KEY = $(TA_DEV_KIT_DIR)/keys/default_ta.pem
```

Comment:
* `?=` instead of `=`  ?
* rename in `TA_SIGN_KEY` as named in `mk/config.in` ?